src/content/App/App.types.ts

```diff
@@ -1,52 +1,51 @@
-import { defaultColorList } from "../../helpers/courseColors";
-
 export enum SectionType {
-    lecture,
-    tutorial,
-    lab
+  lecture,
+  tutorial,
+  lab,
 }
 
 export enum Term {
-    summerOne,
-    summerTwo,
-    summerFull,
-    winterOne,
-    winterTwo,
-    winterFull,
+  summerOne,
+  summerTwo,
+  summerFull,
+  winterOne,
+  winterTwo,
+  winterFull,
 }
 
 export enum Views {
-    calendar,
-    settings
+  calendar,
+  settings,
 }
 
+export const Term_String_Map: { [key in Term]: string } = {
+  [Term.summerOne]: "S1",
+  [Term.summerTwo]: "S2",
+  [Term.summerFull]: "S",
+  [Term.winterOne]: "W1",
+  [Term.winterTwo]: "W2",
+  [Term.winterFull]: "W",
+};
 
-export const Term_String_Map: { [key in Term]: string} = {
-    [Term.summerOne]: 'S1',
-    [Term.summerTwo]: 'S2',
-    [Term.summerFull]: 'S',
-    [Term.winterOne]: 'W1',
-    [Term.winterTwo]: 'W2',
-    [Term.winterFull]: 'W',
-}
+export type SupplementaryData = {
+  instructors: string[];
+  locations: string[];
+};
 
 export type SectionDetail = {
-    term: Term,
-    days: string[]
-    startTime: string,
-    endTime: string,
-    location?: string,
-    dateRange: string   
-}
+  term: Term;
+  days: string[];
+  startTime: string;
+  endTime: string;
+  dateRange: string;
+};
 
 export interface ISectionData {
-    code: string,
-    name: string,
-    type: SectionType,
-    instructors: string[],
-    sectionDetails: SectionDetail[]
-    term: Term,
-    worklistNumber: number,
-    color: string
-    courseID?: string
+  code: string;
+  name: string;
+  sectionDetails: SectionDetail[];
+  term: Term;
```

Comment:
if term is in section detail do we still need it?

Reply:
I tried removing it but it breaks a lot of things since we use it in a lot places. Maybe later we can try and refactor and rework the calendar stuff that uses it